node/pkg/reporter/app.go

```diff
@@ -124,13 +125,19 @@ func (a *App) startReporters(ctx context.Context) {
 	}
 }
 
-func getConfigs(ctx context.Context) ([]Config, error) {
-	reporterConfigs, err := db.QueryRows[Config](ctx, GET_REPORTER_CONFIGS, nil)
+func fetchConfigs() ([]Config, error) {
+	chain := os.Getenv("CHAIN")
+	if chain == "" {
+		log.Info().Str("Player", "Reporter").Msg("CHAIN env not set, defaulting to baobab")
+		chain = "baobab"
+	}
+	endpoint := fmt.Sprintf("https://config.orakl.network/%s_configs.json", chain)
+	configs, err := request.Request[[]Config](request.WithEndpoint(endpoint))
 	if err != nil {
-		log.Error().Str("Player", "Reporter").Err(err).Msg("failed to load reporter configs")
+		log.Error().Str("Player", "Reporter").Err(err).Msgf("failed to fetch configs from endpoint %s", endpoint)
```

Comment:
let's remove error log here, it'll probably be duplicate logging from line 66

Reply:
done!